src/main/java/teammates/ui/controller/InstructorSearchPageAction.java

```diff
@@ -1,68 +0,0 @@
-package teammates.ui.controller;
```

Comment:
I don't think search-related stuffs are part of this PR...

Reply:
yup u r right...thats how i got myself into this conflict...
but search page was already implemented? just wondering y it is still not deleted

Reply:
Not entirely; searching for feedback response comments are still not migrated yet.

Reply:
oh i see
wokie i will roll back then ;_;